Dockerfile

```diff
@@ -9,3 +9,6 @@ RUN pip install --no-cache-dir -r requirements.txt
 EXPOSE 8080
 
 CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8080"]
+
+# Set environment variables
+ENV TEQUILA_API_KEY=${TEQUILA_API_KEY}
```

Comment:
What does `${TEQUILA_API_KEY}` mean?

Reply:
In a Dockerfile, `${VARIABLE_NAME}` is a way to reference environment variables. In this case, `TEQUILA_API_KEY` is an environment variable that will be set when the Docker image is built. The value of this environment variable will be the API key for the Tequila API. 

This is a way to securely pass sensitive information, like API keys, into a Docker container without hardcoding them into the Dockerfile or the application code. The actual value of the `TEQUILA_API_KEY` environment variable should be provided when building the Docker image or when starting a Docker container from the image.

Reply:
Use dotenv instead